index.html

```diff
@@ -5,11 +5,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
     <title>gabriel's portfolio</title>
 </head>
 
 <body>
+    <div class="container">
```

Comment:
- [ ] [OPTIONAL] Its best to use the `main` tag for the main section of your web page